pruning/architecture/pruning_loop.py

```diff
@@ -152,13 +146,13 @@ def prune_model(
     wandb_run: Run,
     checkpoints_interval: Interval,
     device: torch.device,
-    early_stopper: None | utility.training.EarlyStopper = None,
 ) -> pd.DataFrame:
     """Prune the model using the given method.
 
     Args:
```

Comment:
`early_stopper (None | utility.training.EarlyStopper, optional)` should be deleted from docstring.